src/shared/utils/s3.ts

```diff
@@ -22,6 +23,19 @@ export async function deleteS3(bucket: string, key: string): Promise<void> {
   await send(deleteCommand);
 }
 
+export async function fetchS3(
```

Comment:
@NKannan14 `fetchS3` looks like it is doing the same work as the `getS3Object` from the /integration_tests/helpers/s3Helper.ts. Are you happy to refactor the tests to use this? I will also use `fetchS3` in the Verify Transformation lambda.

Reply:
@MelAtGDS  at getObjects3 function is  widely used in integration tests, I am thinking it will be easier to change after Peter's integration test PR and this 276 PR merge but if it essential then I will change in this PR

Reply:
Thanks Nithya, Yes happy to go with your approach. Keep me posted as I'll need it for the Verify PR too :)